mpls-lab/lab_setup.yml

```diff
@@ -40,6 +40,16 @@
     - inventory/gns3_vars.yml
   collections:
     - davidban77.gns3
+  pre_tasks:
+    - name: check env vars
+      set_fact:
+        NETAUTOMATOR_IP: "{{ lookup('env', 'NETAUTOMATOR_IP') }}"
+        NETAUTOMATOR_PWD: "{{ lookup('env', 'NETAUTOMATOR_PWD') }}"
+        NETAUTOMATOR_USR: "{{ lookup('env', 'NETAUTOMATOR_USR') }}"
+    - name: fail if env vars not set
+      fail:
+        msg: "NETAUTOMATOR_IP env var not set. Run `source .env`"
+      when: NETAUTOMATOR_IP == ""
```

Comment:
At the `lab_setup.yml` you only "need" is the `NETAUTOMATOR_IP`, the other ones not so much... So maybe this step needs to be added on the `network_provissioning.yml` playbook.